jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/SlideGroup.java

```diff
@@ -74,14 +79,49 @@ public boolean previousButtonIsActive() {
         return !previousButtonIsDisabled();
     }
 
+    public WebList getAllSlide() {
+        return this.finds(".v-card");
+    }
+
+    @JDIAction("Get position of the '{name}'")
+    public int position() {
+        int index = 0;
+        int position = -1;
+        for (WebElement slideCard : getAllSlide()) {
+            index++;
+            if (slideCard.getAttribute("class").contains("success")
```

Comment:
need to add comment when such condition should be applied. As I see in spec(https://vuetifyjs.com/en/api/v-slide-group/#links) - "#active-class	
string
'v-slide-item--active'
The active-class applied to children when they are activated."

Reply:
removed contains.("success")